AndroidP/README.md

```diff
@@ -0,0 +1,52 @@
+# Android P Spikes
+
+## [Display coutout support](https://developer.android.com/preview/features.html#cutout)
+
+
+Notches are called Display Cutouts on Android and Android P is adding support for them
+
+In Android P you can simulate a display coutout with three different form factors.
+
+![](https://developer.android.com/preview/images/emulator-devoptions-cutout_2x.png)
+
+
+> Tall display cutout
+
+
+When your activity screen goes behind the cutout you can specify how it should behave:
+
+* [`LAYOUT_IN_DISPLAY_CUTOUT_MODE_DEFAULT` ](https://developer.android.com/reference/android/view/WindowManager.LayoutParams.html#LAYOUT_IN_DISPLAY_CUTOUT_MODE_DEFAULT)
+
+> The window is allowed to extend into the DisplayCutout area, only if the DisplayCutout is fully contained within the status bar. Otherwise, the window is laid out such that it does not overlap with the DisplayCutout area.
```

Comment:
Could you indent all these blockquotes by 4 spaces, so they show _inside_ the bullet items instead of outside? Will greatly help readability

Reply:
Didn't know you can do that 💯